lesson-3-state-hooks-and-actions/README.md

```diff
@@ -0,0 +1,22 @@
+# Lesson 8.2 - Working with State
```

Comment:
TODO: update order
8.0 Intro
8.1 Webpack, props, JSX
8.2 - Working with state
8.3 - useEffect / side effects/ pure functions
8.4 - Class based and reusable components